internal/rty/scroll.go

```diff
@@ -408,13 +408,21 @@ func adjustElementScroll(prevInt interface{}, newChildren []string) (*ElementScr
 	next := &clone
 	next.children = newChildren
 
+	if len(newChildren) == 0 {
+		next.elementIdx = 0
+		return next, ""
+	}
 	if len(prev.children) == 0 {
 		sel := ""
 		if len(next.children) > 0 {
 			sel = next.children[0]
 		}
 		return next, sel
 	}
+	if prev.elementIdx >= len(prev.children) {
```

Comment:
I'd probably add a comment along the lines of "this should be impossible, but it happened, so here we are"